stdlib/influxdata/influxdb/to_test.go

```diff
@@ -692,45 +692,6 @@ func TestTo_Process(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "input tables with duplicate group key",
-			spec: &influxdb.ToProcedureSpec{
-				Spec: &influxdb.ToOpSpec{
-					Org:               "my-org",
-					Bucket:            "my-bucket",
-					TimeColumn:        "_time",
-					TagColumns:        []string{"tag1", "tag2"},
-					MeasurementColumn: "_measurement",
-				},
-			},
-			data: []*executetest.Table{
-				{
-					KeyCols: []string{"_field", "_measurement"},
-					ColMeta: []flux.ColMeta{
-						{Label: "_time", Type: flux.TTime},
-						{Label: "_measurement", Type: flux.TString},
-						{Label: "_field", Type: flux.TString},
-						{Label: "_value", Type: flux.TFloat},
-					},
-					Data: [][]interface{}{
-						{execute.Time(11), "m", "f", 1.0},
-					},
-				},
-				{
-					KeyCols: []string{"_field", "_measurement"},
-					ColMeta: []flux.ColMeta{
-						{Label: "_time", Type: flux.TTime},
-						{Label: "_measurement", Type: flux.TString},
-						{Label: "_field", Type: flux.TString},
-						{Label: "_value", Type: flux.TFloat},
-					},
-					Data: [][]interface{}{
-						{execute.Time(11), "m", "f", 1.0},
-					},
-				},
-			},
-			wantErr: errors.New("to() found duplicate table with group key: {_field=f,_measurement=m}"),
-		},
```

Comment:
We no longer get this error I guess due to the chunked implementation?

Reply:
We never got this error because it wasn't strictly possible, but now we don't really have a way to check for it because the chunked implementation expects us to receive table chunks with duplicate group keys.